server/index.js

```diff
@@ -0,0 +1,72 @@
+// const express = require('express');
+import express from 'express';
+import session from 'express-session';
+import morgan from 'morgan';
+
+import bodyParser from 'body-parser';
+
+import graphqlHTTP from 'express-graphql';
+import schema from './schema';
+import db from './db';
+
+import webpack from 'webpack';
+import webpackDevMiddleware from 'webpack-dev-middleware';
+import webpackHotMiddleware from 'webpack-hot-middleware';
+
+const app = express();
+
+const PORT = 3000;
```

Comment:
Now that I thought about this a little more, I find it unlikely that developers will want to change this port, at least when developing this project. On their own server, they definitely would want that capability.